<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QPushButton" name="ask_button">
      <property name="text">
       <string>Ask</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="tokens_label">
      <property name="statusTip">
       <string extracomment="Max Tokens ChatGPT will use. ChatGPT converts each word into a legible token whenever you ask a question."/>
      </property>
      <property name="text">
       <string>Tokens</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="reply_label">
      <property name="text">
       <string>Reply:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="temp_label">
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="statusTip">
       <string extracomment="The setting is &quot;temperature,&quot; and whether you take it literally or metaphorically, the lower the temperature, the more literal the text output. The higher the temperature, the more all rules go out the window.">Temperature</string>
      </property>
      <property name="text">
       <string>Temperature</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="tokens_combobox"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="models_label">
      <property name="statusTip">
       <string extracomment="Choosing a model is equivalent to choosing an AI. Unlike the human brain, models or AIs tend to be highly specialized for a specific set of tasks or inputs.">Models</string>
      </property>
      <property name="text">
       <string>Models</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="models_combobox">
      <property name="toolTip">
       <string>Model that ChatGPT uses. eg. gpt-3.5-turbo</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QTextEdit" name="reply_box">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="question_label">
      <property name="text">
       <string>Question:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="temp_combobox"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="question_box"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
